packages/api/src/ontology/WirePropertyTypes.ts

```diff
@@ -29,6 +29,7 @@ export type WirePropertyTypes =
   | "marking"
   | "numericTimeseries"
   | "stringTimeseries"
+  | "sensorTimeseries"
```

Comment:
This is the new timeseries sensor type I added, I will save the rest of the functionality updates I added for that in the other PR I mentioned